crates/bevy_camera/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "bevy_camera"
+version = "0.8.0-dev"
+edition = "2021"
+description = "Provides camera utilities for Bevy Engine"
+homepage = "https://bevyengine.org"
+repository = "https://github.com/bevyengine/bevy"
+license = "MIT OR Apache-2.0"
+keywords = ["bevy"]
+
+[dependencies]
+# bevy
+bevy_app = { path = "../bevy_app", version = "0.8.0-dev" }
+bevy_asset = { path = "../bevy_asset", version = "0.8.0-dev" }
+bevy_ecs = { path = "../bevy_ecs", version = "0.8.0-dev" }
+bevy_utils = { path = "../bevy_utils", version = "0.8.0-dev" }
+bevy_render = { path = "../bevy_render", version = "0.8.0-dev" }
+bevy_math = { path = "../bevy_math", version = "0.8.0-dev" }
+bevy_core = { path = "../bevy_core", version = "0.8.0-dev" }
+bevy_core_pipeline = { path = "../bevy_core_pipeline", version = "0.8.0-dev" }
```

Comment:
could you reorder them alphabetically and check that they are all needed? I'm not sure for bevy_utils, bevy_math or bevy_core